File: 体験会java中級/Level28.java
package camp.trial02;

public class Level28 {

	public static void main(String[] args) {
		/*
		 * 配列の要素全ての合計値と、その平均値を表示して下さい。
		 */

		int[] points = {10,25,35,40,55};
		int sum = 0;

		for(int point : points) {
			sum += point;
		}

			System.out.println("合計値：" + sum);
			System.out.println("平均値：" + sum/points.length);

	}

}

File: 体験会PHP中級/Level28.php
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>GEEK JOB TRYAL</title>
    </head>
    <body>

        <p>【課題28】</p>
        <p>配列の要素全ての合計値と、その平均値を表示して下さい。</p>
        -------------------------<br>
        <p>【実行結果】</p><br>
        <?php
        // ここから
        $points = array(10,25,35,40,55);

        //追加
        $sum = 0;
        foreach($points as $point) {
          $sum += $point;
        }
        echo $sum;

        // ここまで
        ?>
        <!-- これは消しちゃだめなやつ -->
        <?php include "../footer.php" ?>
    </body>
</html>
